Sources/SecureXPC/Server/XPCServer.swift

```diff
@@ -108,6 +108,10 @@ public class XPCServer {
     public static func forThisXPCService() throws -> XPCServer {
         try XPCServiceServer._forThisXPCService()
     }
+
+    public static func newAnonymousService() -> XPCServer {
```

Comment:
Based on the [Swift.org API Design Guidelines](https://www.swift.org/documentation/api-design-guidelines/) we ought to call this `makeAnonymousService()`.